### PR TITLE
Exchange monitoring

### PR DESCRIPTION
**Pull request by neville-freeman**
Bitmax monitoring

### CODE REVIEW

**Comment by neville-freeman** [comment]
Bitmax alert

File: modules/exchange-monitor/exchange-monitor.py [hunk 1]
```diff
@@ -120,3 +135,9 @@ def process_coinex_assets():
             print("failed to process coinex assets")
             coinex_active.set(False)         
         time.sleep(SLEEP_TIME)
```
Comment (disassembler) [inline modules/exchange-monitor/exchange-monitor.py]: this sleep should be removed.

**Comment by disassembler** [comment]
@neville-freeman this looks good. Please squash the commits into a single commit and force push and I'll approve/merge.

**Pull request merged**